### PR TITLE
feat: initialize router

### PR DESCRIPTION
**Pull request by aisyahnnd**
https://www.loom.com/share/44b5dda38d7146d19d40de10a18c519e

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -3,12 +3,13 @@ body {
 }
 
 #root {
-  max-width: 1280px;
+  width: 1280px;
   margin: 0 auto;
   /* padding: 2rem; */
```
Comment (argorfc) [inline src/App.css]: Please remove unused line.

File: src/App.css [hunk 2]
```diff
@@ -3,12 +3,13 @@ body {
 }
 
 #root {
-  max-width: 1280px;
+  width: 1280px;
   margin: 0 auto;
```
Comment (argorfc) [inline src/App.css:7]: Please remove unused line.

File: src/App.jsx [hunk 3]
```diff
@@ -5,19 +5,46 @@ import { AllTrainingEventTable, MyTrainingEventTable } from './Parts';
...
+        />
+        <Route
+          path="/training/:id"
+          element={<TrainingDetailPage />}
+        />
+        <Route
+          path="/training/create"
+          element={<TrainingCreatePage />}
+        />
+
```
Comment (argorfc) [inline src/App.jsx]: Please remove space.

**Review by argorfc — CHANGES_REQUESTED** [review]
Read the comments!

**Comment by aisyahnnd** [comment]
Already fix: d2f5020d832d77294b4774f776b021232952edf0

**Review by argorfc — APPROVED** [review]
Ready to merge.

**Pull request merged**